src/screens/QRPresentationScreen/index.tsx

```diff
@@ -0,0 +1,53 @@
+import {FC, useEffect, useState} from 'react';
+import {Dimensions} from 'react-native';
+import {SSIBasicContainerSecondaryStyled as SSIContainer} from '../../styles/components';
+import {QRContainer, QRPresentationViewContainer} from 'src/styles/components/screens/QRPresentationScreen';
+import {CreateElementArgs, QRType} from '@sphereon/ssi-sdk.qr-code-generator';
+import {agentContext} from '../../agent';
+
+const {width} = Dimensions.get('screen');
+
+type QRPresentationScreenProps = {};
```

Comment:
for now you can just not add any props, so this type can be removed. we can add it once we need it

Reply:
props have been removed